swift/Samples/ModalContainer/Sources/ModalContainerViewController.swift

```diff
@@ -0,0 +1,344 @@
+                    modal.dimmingView?.alpha = 0
+                },
+                completion: { _ in
+                    modal.viewController.view.removeFromSuperview()
+                    modal.viewController.removeFromParent()
+                    modal.dimmingView?.removeFromSuperview()
+                }
+            )
+        }
+
+        for modal in screensNeedingAppearanceTransition {
+            let displayInfo = ModalDisplayInfo(containerSize: view.bounds.size, style: modal.style, animated: modal.animated)
+            modal.viewController.view.frame = displayInfo.incomingInitialFrame
+            modal.viewController.view.transform = displayInfo.incomingInitialTransform
+            modal.viewController.view.alpha = displayInfo.incomingInitialAlpha
+
+            UIView.animate(
+                withDuration: displayInfo.duration,
+                delay: 0.0,
+                options: displayInfo.animationOptions,
+                animations: {
+                    modal.viewController.view.bounds = CGRect(
+                        origin: .zero,
+                        size: displayInfo.frame.size
+                    )
+                    modal.viewController.view.center = CGPoint(
+                        x: displayInfo.frame.midX,
+                        y: displayInfo.frame.midY
+                    )
+                    modal.viewController.view.transform = displayInfo.transform
+                    modal.viewController.view.alpha = displayInfo.alpha
+                    modal.dimmingView?.alpha = 1
+                },
+                completion: { _ in
+                    UIAccessibility.post(notification: .screenChanged, argument: nil)
+                }
+            )
+        }
+
+        // Update our state to reflect the new screens post-update
+        presentedScreens = newScreens
+
+        // Sort our views. We go front to back to allow dismissed views to appear above currently presented views (this will matter after transition support is added).
+        for modal in presentedScreens.reversed() {
+            view.sendSubviewToBack(modal.viewController.view)
+            if let dimmingView = modal.dimmingView {
+                view.sendSubviewToBack(dimmingView)
+            }
+        }
+
+        view.sendSubviewToBack(baseScreenViewController!.view)
+
+        setNeedsStatusBarAppearanceUpdate()
+
+        if #available(iOS 11.0, *) {
+            setNeedsUpdateOfHomeIndicatorAutoHidden()
+            setNeedsUpdateOfScreenEdgesDeferringSystemGestures()
+        }
+
+        // Set the topmost screen to be the accessibility modal
+        presentedScreens.last?.viewController.view.accessibilityViewIsModal = true
+        for modal in presentedScreens.dropLast() {
+            modal.viewController.view.accessibilityViewIsModal = false
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        if baseScreenViewController == nil {
```

Comment:
@dhavalshreyas moved this to viewDidLoad. ref [PR](https://github.com/square/workflow/pull/1087).